app/services/return-requirements/fetch-return-requirements.service.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+
+/**
+ * Fetches existing return requirements needed for `/return-requirements/{sessionId}/check` page
+ * @module FetchReturnRequirementsService
+ */
+
+const ReturnVersionModel = require('../../models/return-version.model.js')
+
+/**
+ * Fetches existing return requirements needed for `/return-requirements/{sessionId}/check` page
+ *
+ * @param {string} licenceId - The UUID for the licence to fetch
+ *
+ * @returns {Promise<Object>} The existing return requirements for the matching licenceId
+ */
+async function go (licenceId) {
+  const data = await _fetchReturnRequirements(licenceId)
+
+  return data
+}
+
+async function _fetchReturnRequirements (licenceId) {
+  console.log(licenceId)
+  const result = await ReturnVersionModel.query()
+    .where('licenceId', licenceId)
+    .withGraphFetched('returnRequirementPoints')
+    .modifyGraph('returnRequirementPoints', (builder) => {
```

Comment:
By default 'withGraphFetched' will fetch all columns for the related model, unless you explicitly say otherwise. So I think lines 28 to 31 can be removed. An example where Alan has done this `test/models/address.model.test.js` line 69. (Granted it is a test file though).

Reply:
I actually don't need all of the columns so i've changed it to just the id